src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/RazorRequestContext.cs

```diff
@@ -6,49 +6,19 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
 
-internal readonly struct RazorRequestContext
+internal readonly struct RazorRequestContext(VersionedDocumentContext? documentContext, IRazorLogger logger, ILspServices lspServices, string method, Uri? uri)
 {
-    public readonly VersionedDocumentContext? DocumentContext;
-    public readonly IRazorLogger Logger;
-    public readonly ILspServices LspServices;
-
-#if DEBUG
-    public readonly string? LspMethodName;
-    public readonly Uri? Uri;
-#endif
-
-    public RazorRequestContext(
-        VersionedDocumentContext? documentContext,
-        IRazorLogger logger,
-        ILspServices lspServices)
-    {
-        DocumentContext = documentContext;
-        LspServices = lspServices;
-        Logger = logger;
-    }
-
-#if DEBUG
-    public RazorRequestContext(
-        VersionedDocumentContext? documentContext,
-        IRazorLogger logger,
-        ILspServices lspServices,
-        string lspMethodName,
-        Uri? uri) : this(documentContext, logger, lspServices)
-    {
-        LspMethodName = lspMethodName;
-        Uri = uri;
-    }
-#endif
+    public readonly VersionedDocumentContext? DocumentContext = documentContext;
+    public readonly IRazorLogger Logger = logger;
+    public readonly ILspServices LspServices = lspServices;
+    public readonly string Method = method;
+    public readonly Uri? Uri = uri;
```

Comment:
do you know why this (`LspMethodName` and `Uri`) were guarded by `#if DEBUG` before?

Reply:
I felt like it at the time :)